table_connector.py

```diff
@@ -257,3 +258,14 @@ def get_cleaned_fully_merged_messages():
     print 'Printing first row of address book with full name column (merged first name/last name/company):'
     display(address_book_df.head(1))
     return fully_merged_messages_df, address_book_df
+
+if __name__=="__main__":
+    messages, addresses = get_cleaned_fully_merged_messages()
+    if len(sys.arv)>1:
+        file_path = sys.argv[1]
+        messages.to_csv(file_path+"messages.csv")
+        addresses.to_csv(file_path+"addresses.csv")
+    else:
+        print messages
```

Comment:
Point well made.  Perhaps just drop a to-do to support additional flags and thus an option for this.